### PR TITLE
Remove non-frac grid coupled tests (Replacement PR, was #584)

### LINKED ISSUE #581: convert 4 existing cpld_control tests to fractional grid; drop existing non-frac RT tests for S2S, S2SW
## Description

The fractional grid implementation for S2S,S2SW is the default going forward. It has been well-tested in ufs-weather since January, 2021. We should drop any non-fractional grid coupled tests. 

## Solution

Convert the ``cpld_2threads``, ``cpld_decomp``, ``cpld_satmedmf`` and ``cpld_ca`` tests to use the fractional grid. This will change the baselines for these tests. 

Drop the following tests:

```
cpld_control
cpld_restart
cpld_control_c192
cpld_restart_c192
cpld_control_c384
cpld_restart_c384
cpld_bmark
cpld_restart_bmark
cpld_bmark_wave
cpld_debug
```

### PR DESCRIPTION
**Pull request by DeniseWorthen**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

- makes ``nems_frac`` the default CPLMODE
- sets ``FRAC_GRID=true`` as the default
- removes non-frac grid coupled regression tests; drops identifier "frac" from remaining test names
- reduces wall clock for cpld_debug. This test runs in about 5 mins but wall clock was set at 60mins.
- reduces cpld_control_c384 to 12 hour forecast; modifies corresponding restart test
- removes non-v16 benchmark and non-frac 35d tests 
- comments out cpld_2threads test (Issue #622)

### Issue(s) addressed

- fixes #581 

## Testing

Pre-commit testing for gaea.intel shows that the following tests do not reproduce:

```
cpld_ca
cpld_satmedmf
cpld_decomp
cpld_2threads
cpld_control_wave
```

This is the expected result since these are the low-resolution tests which are currently running with non-frac grid. All other coupled tests pass the current develop branch comparison (develop-20210513).

A test baseline was created for the new suite of coupled tests for gaea.intel and all test passed against the test baseline.

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3

## Dependencies

None

### CODE REVIEW

**Comment by BrianCurtis-NOAA** [comment]
Machine: hera
Compiler: intel
Job: BL
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/663654892/20210607153016/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_93608
Test cpld_control_wave 009 failed failed 
Test cpld_control_wave 009 failed in run_test failed 
Please make changes and add the following label back:
hera-intel-BL

**Comment by DeniseWorthen** [comment]
I thought I had fixed this---I need to add back the FV3_GFS_2017_coupled to the S2SW compile line. All other baselines completed.

**Review by JessicaMeixner-NOAA — APPROVED** [review]

File: tests/rt_35d.conf [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2012100100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013010100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013040100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013070100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013100100
+#COMPILE | APP=S2SW SUITES=FV3_GFS_v16_coupled,FV3_GFS_v16_coupled_noahmp,FV3_GFS_v16_coupled_nsstNoahmp                   | + hera.intel orion.intel | fv3 |  |
```
Comment (climbfuji) [inline tests/rt_35d.conf:5]: Maybe I missed this in the discussion today, why are these tests commented out and not removed?

**Review by climbfuji — APPROVED** [review]
Very much appreciate to remove so many tests and to shorten some of the other coupled tests.

File: tests/rt_35d.conf (hunk 1 above)
Comment (DeniseWorthen) [reply tests/rt_35d.conf:5]: I retained them because we are still running the RT version of the benchmark v16 configuration (``cpld_bmark_wave_v16``).

**Comment by BrianCurtis-NOAA** [comment]
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/663654892/20210607134509/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_164531
Baseline creation and move successful
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/663654892/20210607144956/ufs-weather-model
Please manually delete: /work/noaa/stmp/bcurtis/stmp/bcurtis/FV3_RT/rt_436940
Test datm_mx025_cfsr 080 failed in check_result failed 
Test datm_mx025_cfsr 080 failed in run_test failed 
Test cpld_restart_bmark_v16 010 failed in check_result failed 
Test cpld_restart_bmark_v16 010 failed in run_test failed 
Please make changes and add the following label back:
orion-intel-BL

**Comment by DeniseWorthen** [comment]
No obvious reason why the 2 orion tests failed. I'll re-run them.

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**